gradle.properties

```diff
@@ -65,7 +65,7 @@ versioningPluginVersion=1.1.0
 # will be downloaded and used. If not set, the existing local installations will be use
 # The version of npm corresponds to the given version of node
 npmVersion=6.14.13
```

Comment:
We tend to try and keep the `node` and `npm` versions linked as these are often paired together across major releases. See https://nodejs.org/en/download/releases/ for a mapping.

This isn't to say we couldn't unlink them but it's not a configuration that is normally installed when installing Node.